flipt-client-go/evaluation.go

```diff
@@ -1,12 +1,14 @@
 package evaluation
 
 /*
-#cgo darwin,arm64 LDFLAGS: -L./ext/darwin_arm64 -lfliptengine -Wl,-rpath -Wl,./ext/darwin_arm64
-#cgo linux,arm64 LDFLAGS: -L./ext/linux_arm64 -lfliptengine -Wl,-rpath -Wl,./ext/linux_arm64
-#cgo linux,amd64 LDFLAGS: -L./ext/linux_x86_64 -lfliptengine -Wl,-rpath -Wl,./ext/linux_x86_64
+#cgo CFLAGS: -I./ext
+#cgo darwin,arm64 LDFLAGS: -L${SRCDIR}/ext/darwin_arm64 -lfliptengine  -Wl,-rpath,${SRCDIR}/ext/darwin_arm64
+#cgo darwin,amd64 LDFLAGS: -L${SRCDIR}/ext/darwin_x86_64 -lfliptengine  -Wl,-rpath,${SRCDIR}/ext/darwin_x86_64
```

Comment:
```suggestion
```
we dont currently support Mac x64

Reply:
@markphelps  nice joke :) All my changes for Flipt were made on macOSX 10.13 with Intel Core i5.

Reply:
yeah i just mean that the engine isn't actually packaged to support darwin intel, although we certainly could, it just isn't currently:

https://github.com/flipt-io/flipt-client-sdks/actions/runs/8457157632